dpnp/dpnp_algo/dpnp_elementwise_common.py

```diff
@@ -24,89 +24,546 @@
 # THE POSSIBILITY OF SUCH DAMAGE.
 # *****************************************************************************
 
-
-import dpctl.tensor as dpt
+import numpy
+from dpctl.tensor._elementwise_common import (
+    BinaryElementwiseFunc,
+    UnaryElementwiseFunc,
+)
 
 import dpnp
-import dpnp.backend.extensions.vm._vm_impl as vmi
 from dpnp.dpnp_array import dpnp_array
 from dpnp.dpnp_utils import call_origin
 
 __all__ = [
```

Comment:
The code coverage is only 59% per that file. The missing lines:
> 147-148, 153-154, 160, 165-166, 172-173, 180-183, 318-319, 324-325, 331, 336-337, 344, 351-354, 452, 458, 529, 540, 550, 587-631

Can we increase the reporting coverage rate>